benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py

```diff
@@ -228,7 +228,7 @@ def __get_metadata(self, kind: str = None, database: str = None, status: str = N
         if run_artifacts_url:
             metadata.update({'run_artifacts_url': run_artifacts_url})
         if database:
-            metadata.update({'vm_os_version': 'centos8'})
+            metadata.update({'vm_os_version': 'centos-stream8'})
```

Comment:
This should really be fully qualified with the repo (I presume it's docker.io, which has its own issues in terms of throttling).  Likewise for everything else.  At a minimum, it needs to be cleaned up when you get a chance.

Reply:
Full path [here](https://github.com/redhat-performance/benchmark-runner/blob/b7858ee7f63aa56ddf13e582beabf5d4d8c2b6bf/benchmark_runner/common/template_operations/templates/hammerdb/internal_data/hammerdb_vm_template.yaml#L80)

Reply:
OK